rrbot_description/urdf/rrbot.xacro

```diff
@@ -30,14 +30,14 @@
     <collision>
       <origin xyz="0 0 ${height1/2}" rpy="0 0 0"/>
       <geometry>
-	<box size="${width} ${width} ${height1}"/>
+	       <box size="${width} ${width} ${height1}"/>
```

Comment:
Why is this indented by 7 spaces? The rest of file appears to use 2 space indent, we should keep consistent.